lib/controllers/messages.ts

```diff
@@ -401,18 +425,21 @@ export function CreateMessageHandler(
     // emit created message event to the output queue
     //
 
-    const messageContent: MessageContent = {
-      markdown: messagePayload.content.markdown,
-      subject: messagePayload.content.subject
+    const newMessageWithContent: NewMessageWithContent = {
+      ...newMessageWithoutContent,
```

Comment:
This is a "big" change, as we pass the whole MessageWithContent here with this PR 
(actually we pass only some properties of the message object + messageContent)

Reply:
actually we pass only the message.id + message.content while now we pass the whole message to the email_notification_queue_handler (as it need to check ttl and createdAt):

https://github.com/teamdigitale/digital-citizenship-functions/pull/158/files#diff-369872c88999ee654209f0771aaa8f25L124